src/Weather.js

```diff
@@ -18,17 +18,20 @@ class Weather extends CityDisplay {
   }
 
 
-  handleWeatherSubmit = e => {
+  handleWeatherSubmit = (e) => {
     e.preventDefault();
 
     // todo: add query entry to weatherUrl
 
+    //todo: move `get`s to app.js?
     
-    let weatherUrl = 'http://localhost/3003/weather&q=';
+    let weatherUrl = `http://localhost/3001/weather?lat=${lat}&lon=${lon}`;
```

Comment:
Good work so far!

- For this line, there is an error in the url
-  We can also add the address to server in the .env file and then use that env variable here, passing it into the url after `/weather`.